.github/security.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+**Do not open up a GitHub issue if the bug is a security vulnerability in Rails**.
+Instead to refer to our [security policy](https://rubyonrails.org/security/).
+
+## Supported Versions
+
+| Version | Supported (major issues) | Supported (minor issues) |
+| ------- | ------------------------ | ------------------------ |
+| 6.0.x   | :white_check_mark:       | :white_check_mark:       |
+| 5.2.x   | :white_check_mark:       | :white_check_mark:       |
+| < 5.2   | :x:                      | :x:                      |
```

Comment:
I get the point about adding the table here, but what about just linking to our maintenance policy so there's one less place to change?

Reply:
Done!